### PR TITLE
Completion of usage examples for write_line_int

### PR DESCRIPTION
**Pull request by SimonRhook**
# Overview
Added C\# OOP and Python Language versions. Updated name for top level C\# version to properly fit naming convention and fixed a number within the top level varient to properly match the output.

Splashkit Function: `write_line_int`
- [ x] Title and explanation (.txt)
- [x ] C++ code
- [x ] C# code (top-level statements)
- [x ] C# code (Object-Oriented Programming)
- [ x] Python code
- [x ] Screenshot

# Usage Example Checks
- [x ] Simple, clear demonstration of the function
- [ x] Code uses Splashkit functions
- [ x] Tested in Chrome and Firefox

### CODE REVIEW

**Review by ShaunR1991 — CHANGES_REQUESTED** [review]
Good first contribution. If you could just make the changes to the OOP version of the C# code we should be good to go.

File: public/usage-examples/terminal/write_line_int/write_line_int-1-simple-oop.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            WriteLine(-165746);
+
+            // Example 3: Print integer after calculation
+            int a = 222-111;
+            int b = 10 * 12;
+            int c = 100 / 5;
+
+            WriteLine(a - b);
+            WriteLine(b);
+            WriteLine(c);
```
[the remark this answers is not in the document]
Comment (SimonRhook) [reply public/usage-examples/terminal/write_line_int/write_line_int-1-simple-oop.cs]: This should now be updated

[the remark this answers is not in the document]
Comment (ShaunR1991) [reply public/usage-examples/terminal/write_line_int/write_line_int-1-simple-oop.cs]: Looks good!

**Comment by breezy-codes** [comment]
I've reviewed this and the code looks good, matches in correctly with how the other code was as well.
However, you will need to rebase this PR to remove the other commits Olivia has done.
There is a troubleshooting page on doing this -
https://thoth-tech.netlify.app/products/splashkit/splashkit-website/onboarding/02-troubleshooting/#how-to-rebase-your-branch

**Pull request closed without merging**